src/components/TestServices.tsx

```
@@ -0,0 +1,60 @@
+        onChange={(e) => setTokenInput(e.target.value)}
+      />
+
+      <input
+        type="text"
+        placeholder="Enter organization name"
+        value={orgInput}
+        onChange={(e) => setOrgInput(e.target.value)}
+        style={{ marginLeft: "10px" }}
+      />
+
+      <button onClick={testFlow} style={{ marginLeft: "10px" }}>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)